ludwig/automl/automl.py

```diff
@@ -142,6 +143,7 @@ def auto_train(
     return train_with_config(dataset, config, output_directory=output_directory, random_seed=random_seed, **kwargs)
 
 
+@DeveloperAPI
```

Comment:
Do we feel like these should potentially be PublicAPI?

Reply:
You are right, also `auto_train` should be PublicAPI as well.  They are referenced in the AutoML user guide:  https://ludwig.ai/latest/user_guide/automl/